xml/installation-installation-ironic_oneview_integration.xml

```diff
@@ -196,19 +195,14 @@ $ cd ~/scratch/ansible/next/ardana/ansible/
 | ID                                   | Name                     |
 +--------------------------------------+--------------------------+
 | c61da588-622c-4285-878f-7b86d87772da | cirros-0.3.4-x86_64      |
-| 6616ef6a-ee7a-478c-b1bf-4c93088a123f | ir-deploy-iso-HOS5.0     |
-| 633d379d-e076-47e6-b56d-582b5b977683 | ir-deploy-kernel-HOS5.0  |
-| d5828785-edf2-49fa-8de2-3ddb7f3270d5 | ir-deploy-ramdisk-HOS5.0 |
-| d6b5d971-42fd-492b-b33e-6e5f2d88e942 | Ubuntu Trusty 14.04 BIOS |
 +--------------------------------------+--------------------------+</screen>
     <para>
-     Ironic deploy images ("ir-deploy-*") were created automatically by HOS
+     Ironic deploy images ("ir-deploy-*") are created automatically by the
      installer. We will be using
      <emphasis role="bold">agent_pxe_oneview</emphasis> Ironic driver which
      needs <emphasis role="bold">ir-deploy-kernel</emphasis> and
-     <emphasis role="bold">ir-deploy-ramdisk</emphasis> images. Managed node
-     will be deployed using <emphasis role="bold">Ubuntu Trusty 14.04
-     BIOS</emphasis> image.
+     <emphasis role="bold">ir-deploy-ramdisk</emphasis> images.
+     The image used to deploy managed nodes will also be shown.
```

Comment:
Mhm ... As it stands, this sounds like a non-information. Which image will it be?

Reply:
Should be

Ironic deploy images (boot image, ir-deploy-kernel, ir-deploy-ramdisk, ir-deploy-iso) are created automatically. The <emphasis role="bold">agent_pxe_oneview</emphasis> Ironic driver requires <emphasis role="bold">ir-deploy-kernel</emphasis> and <emphasis role="bold">ir-deploy-ramdisk</emphasis> images.

Reply:
fixed